tests/trainer/test_scale_schedule.py

```diff
@@ -85,7 +86,8 @@ def test_epochs_scaled(self, ssr: float, use_algorithm: bool, composer_trainer_h
             composer_trainer_hparams.scale_schedule_ratio = ssr
         trainer = composer_trainer_hparams.initialize_object()
 
-        assert trainer.state.max_epochs == int(10 * ssr)
+        assert trainer.state.max_duration.unit == TimeUnit.EPOCH
```

Comment:
Add a test here to test that `1ep` is also scaled correctly by SSR (i.e. your change in the `_convert_time` function?